ddtrace/settings/integration.py

```diff
@@ -86,18 +86,20 @@ def is_header_tracing_configured(self):
         """
         return self.http.is_header_tracing_configured or self.global_config.http.is_header_tracing_configured
 
-    def header_is_traced(self, header_name):
```

Comment:
same here

Reply:
I added back the public method so we do not break the compatibility.